website/docs/language/resources/provisioners/connection.mdx

```diff
@@ -7,37 +7,35 @@ description: >-
 
 # Provisioner Connection Settings
 
-Most provisioners require access to the remote resource via SSH or WinRM, and
+Most provisioners require access to the remote resource via SSH or WinRM and
 expect a nested `connection` block with details about how to connect.
 
--> **Note:** Provisioners should only be used as a last resort. For most
-common situations there are better alternatives. For more information, see
-[the main Provisioners page](/language/resources/provisioners).
+~> **Important:** You should only use provisioners as a last resort. There are better alternatives for most situations. Refer to
+[Declaring Provisioners](/language/resources/provisioners/syntax) for more details.
```

Comment:
Your edit here makes me consider a further tightening:

> **Important:** Use provisioners only as a last resort. There are better alternatives for most situations. [etc, etc]

I dunno if that maybe reads a little _too_ direct/"rude", but it feels easier to scan, at least. :man_shrugging: I'm happy either way, but thought I might as well suggest it since I was here anyway. :grinning:

Reply:
YES. I'm going to do that now. Great call, Martin!